src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -13,37 +12,76 @@ import "../interfaces/ISlasher.sol";
  * @notice This storage contract is separate from the logic to simplify the upgrade process.
  */
 abstract contract DelegationManagerStorage is IDelegationManager {
-    /// @notice Gas budget provided in calls to DelegationTerms contracts
-    uint256 internal constant LOW_LEVEL_GAS_BUDGET = 1e5;
+    // TODO: documentation
+    struct StakerDelegation {
+        address staker;
+        address operator;
+        uint256 nonce;
+        uint256 expiry;
+    }
+
+    // TODO: documentation
+    struct DelegationApproval {
+        address staker;
+        address operator;
+        uint256 nonce;
+        uint256 expiry;
+    }
+
+    // TODO: add constants to interface
 
     /// @notice The EIP-712 typehash for the contract's domain
     bytes32 public constant DOMAIN_TYPEHASH =
         keccak256("EIP712Domain(string name,uint256 chainId,address verifyingContract)");
 
-    /// @notice The EIP-712 typehash for the delegation struct used by the contract
-    bytes32 public constant DELEGATION_TYPEHASH =
-        keccak256("Delegation(address delegator,address operator,uint256 nonce,uint256 expiry)");
+    /// @notice The EIP-712 typehash for the StakerDelegation struct used by the contract
+    bytes32 public constant STAKER_DELEGATION_TYPEHASH =
+        keccak256("StakerDelegation(address staker,address operator,uint256 nonce,uint256 expiry)");
 
-    /// @notice EIP-712 Domain separator
-    bytes32 public DOMAIN_SEPARATOR;
+    /// @notice The EIP-712 typehash for the DelegationApproval struct used by the contract
+    bytes32 public constant DELEGATION_APPROVAL_TYPEHASH =
```

Comment:
*DelegationApproval

Reply:
is this meant to be a comment about a typo? If so, I'm not seeing the typo

Reply:
the struct inside of the quotes, sorry wrong line

Reply:
ohhhh good catch, just fixed this in https://github.com/Layr-Labs/eigenlayer-contracts/pull/97/commits/599b96de577503e67cc45dc0e29aa1b5ae218c0a
ty!